include/CLI/impl/Formatter_inl.hpp

```diff
@@ -137,7 +142,7 @@ CLI11_INLINE std::string Formatter::make_footer(const App *app) const {
     if(footer.empty()) {
         return std::string{};
     }
-    return footer + "\n";
+    return "\n" + footer + "\n";
```

Comment:
Do you think you could pop out this part of the change into a quick PR? This can go into a patch release if you do.

Reply:
I think I can make all fixes over the weekend.
Will it be quickly enough? :)
Or changes in `usage` not getting through anyway?

Reply:
I'd like to make a quick bug fix release with no new features, but the spacing on the footer is close enough to a bug I think it could be included. The usage is a new features, and needs to sit in main a little bit, be added to the readme with a new feature emoji, etc.

Reply:
So the footer change can go into 2.3.1, while the usage will go into 2.4.0.

Reply:
Well, this will be a lesson for me to cut such things in the future :)
In that case, I close this PR and create two separate ones?

Reply:
You can just make the PR for the fix, this one can easily be rebased. I think you can just make a new branch and checkout the related changes from this one.

Reply:
Thanks! Merged. It is usually best to try to keep PRs very focused. :)